src/components/Router/HistoryRouter.tsx

```diff
@@ -36,6 +37,30 @@ const formattedError = ({ response, status }: ParsedError): FormattedError => {
   return { type, message }
 }
 
+export const HistoryRouterWrapper = ({
+  useSteps,
+  fallback,
+  ...props
+}: HistoryRouterWrapperProps) => {
+  const { steps, error, loading, ...useStepsProps } = useSteps()
+
+  if (!steps || loading) {
+    return <Fragment>{fallback}</Fragment>
+  }
+
+  if (error) {
+    return (
+      <div>
+        <p>There was a server error!</p>
+        <p>{error}</p>
+        <p>Please try reloading the app, and try again.</p>
+      </div>
+    )
+  }
+
+  return <HistoryRouter {...props} {...useStepsProps} steps={steps} />
+}
+
 export const HistoryRouter = (props: HistoryRouterProps) => {
```

Comment:
```suggestion 
const HistoryRouter = (props: HistoryRouterProps) => {
```
It looks like it's not used outside this file now ?

Reply:
Going to merge this as it's holding up the release. This can be fixed if needed in a new PR

Reply:
I will add this in an other PR